tests/dotnet.buildtools.prereqs.docker.tests/ManifestTests.cs

```diff
@@ -0,0 +1,79 @@
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace dotnet.buildtools.prereqs.docker.tests;
```

Comment:
Suggestions for namespace / project name:

- `Microsoft.DotNet.BuildToolsPrereqs.Docker.Tests`
- `DotNet.BuildToolsPrereqs.Docker.Tests`

Reply:
Changed to DotNet.BuildToolsPrereqs.Docker.Tests